src/components/profiles/address-views/AuthorCard.tsx

```diff
@@ -53,21 +53,20 @@ export default function AuthorCard({
         </div>
       }
       subtitle={<DfMd source={profile?.content?.about} className='ColorCurrentColor' />}
-      actions={
-        <div className='d-flex align-items-center'>
-          {withTipButton && !isMy && <Donate recipientAddress={address.toString()} />}
-          {isMy && profile && (
-            <ButtonLink
-              href={'/[spaceId]/edit'}
-              as={editSpaceUrl(profile.struct)}
-              className='bg-transparent'
-            >
-              <EditOutlined /> Edit
-            </ButtonLink>
-          )}
-          <FollowAccountButton address={address} />
-        </div>
-      }
+      buttons={[
+        withTipButton && !isMy && <Donate key='donate' recipientAddress={address.toString()} />,
+        isMy && profile && (
```

Comment:
Not trying to be a grammar nazi here, but at a glance it's hard to understand what does `isMy` mean. What does it refer to? Would my `isMyAddress` be a more suitable name?

Reply:
Renamed